pentaho-mql-editor/src/org/pentaho/commons/metadata/mqleditor/editor/controllers/MainController.java

```diff
@@ -99,9 +99,8 @@ public void init() {
    }
 
    public void showDialog() {
-
       dialog.show();
-
+      conditionsTable.update();
```

Comment:
I don't think so because conditionsTable object is created as a dialog object in createBindings method.